packages/server/src/Hocuspocus.ts

```diff
@@ -286,21 +298,30 @@ export class Hocuspocus {
   /**
    * Destroy the server
    */
-  async destroy(): Promise<any> {
-    this.server?.httpServer?.close()
+  destroy(): Promise<any> {
+    return new Promise(async resolve => {
 
-    try {
-      this.server?.webSocketServer?.close()
-      this.server?.webSocketServer?.clients.forEach(client => {
-        client.terminate()
-      })
-    } catch (error) {
-      console.error(error)
-    }
+      this.server?.httpServer?.close()
```

Comment:
I think `close` should take a callback and the promise shouldn't resolve until the callback is called.

Reply:
hmm yep, didnt check this before actually. Will note to change this, as the behaviour has been like this forever